WcaOnRails/app/models/timestamp.rb

```diff
@@ -2,4 +2,8 @@
 
 class Timestamp < ApplicationRecord
   self.primary_key = "name"
+
+  def not_after?(other_date)
```

Comment:
I see you discussing a name, but I can't tell what the original name was. Lars says the new name is to deal with nil, but it's not entirely clear to me how this deals with nil (was my original guess correct?)